app/pages/cif/projects.tsx

```diff
@@ -88,9 +100,11 @@ export function Projects({ preloadedQuery }: RelayProps<{}, projectsQuery>) {
         orderByPrefix: "OPERATOR_BY_OPERATOR_ID__TRADE_NAME",
       }),
       new TextFilter("Proposal Reference", "proposalReference"),
-      new TextFilter("Status", "status", {
-        orderByPrefix: "PROJECT_STATUS_BY_PROJECT_STATUS_ID__NAME",
-      }),
+      new SearchableDropdownFilter(
+        "Status",
+        "status",
+        allProjectStatuses.edges.map((e) => e.node.name)
+      ),
```

Comment:
- Could we add a configuration item to this, that turns on or off the `freeSolo` option of the Autocomplete in the `SerarchableDropdownFilter` component (for statuses, it doesn't make sense to select multiple / by string search)
- Could we have the `SearchableDropdownFilter` hide that 'X' button, since that's already something that the 'Clear' button does on the right hand side? You'll need to set the `disableClearable` option (see [docs](https://mui.com/material-ui/api/autocomplete/#props))